src/lib/model.ts

```diff
@@ -26,10 +42,12 @@ export class Course {
 	description: string
 	duration: number
 	learningOutcomes: string
-	price: number | string = this.collatePrice() || 'Free'
 
 	modules: Module[]
 
+	audiences: Audience[]
+	audience?: Audience
```

Comment:
is that we don't have `audience?: Audience` anymore? (at least in `lpg-management` model)